velox/exec/HashBuild.cpp

```diff
@@ -735,10 +735,14 @@ bool HashBuild::finishHashBuild() {
         }
       }
 
-      const bool hasOthers = !otherTables.empty();
+      // TODO: we need to disable parallel join after
+      // https://github.com/facebookincubator/velox/issues/3567 is fixed.
+      const bool allowParallelJoin =
```

Comment:
allowParallelJoin - this name is confusing; I believe the parallelism applies only to building hash table, no?

Reply:
Yeah, thanks for pointing out. Updated.